MySQLOO/source/LuaObjectBase.cpp

```diff
@@ -153,16 +153,25 @@ void LuaObjectBase::runFunction(GarrysMod::Lua::ILuaBase* LUA, int funcRef, cons
 	va_end(arguments);
 }
 
+static int BuildErrorStack(lua_State *state) {
+	GarrysMod::Lua::ILuaBase* LUA = state->luabase;
+
+	LUA->PushSpecial(GarrysMod::Lua::SPECIAL_GLOB);
+	LUA->GetField(-1, "debug");
```

Comment:
what happens if debug or debug.traceback are nil? I am not sure if we should be required to care for these circumstances but maybe it'd be good to anyways?

Reply:
i've made it check if it's a function, and if it's not it will just return the regular error without stack

Reply:
i wish i could do more however i don't think i can't without linking to something that gets lua_Debug stuff.

Reply:
Yeah that's what I had in mind, you should do the same for the debug part but I'm probably just being paranoid here.

Reply:
if debug is gone, then i believe default gmod will break

if you really want i can add that, but it's up to you